documentation/basics/dataselection.md

```diff
@@ -120,6 +154,8 @@ Widely used image formats such as
 can be imported directly into a project, as well as NumPy binary array
 files with extension `npy`.
 
+OME-Zarr and Neuroglancer Precomputed are only supported for [multiscale](#multiscale) datasets.
```

Comment:
I'm getting hung up on the "only" here. It sounds like somehow the support for these files is worse than for the ones mentioned above, but that is not the case. Could you maybe add a new heading "Multiscale data formats" (like "Widely used image formats such as..." and list them, maybe with a hint that "Add multiscale dataset" has to be used to load those.